### ISSUE TITLE
Support webpack customization

### ISSUE THREAD

**Issue by sessionboy** [lucasecdb/casterly#362]
When Casterly's own webpack configuration cannot meet the demand, we need to expand it.

For example, add a new babel plugin, add alias, etc.

Similar to next.js, we need a configuration file, for example: `casterly.config.js`.

```js
const path = require("path");

module.exports = {
   webpack: (config, { isServer, webpack }) =>{     
      config.resolve.alias['@components'] = path.resolve(__dirname,"src/components");
      config.resolve.alias['@utils'] = path.resolve(__dirname,"src/utils");
      return config;
   },
   devServer:{
      port: 8081
   }
}
```

**Comment by lucasecdb** [comment]
absolutely! I'll try to tackle this over the next week.

**Comment by sessionboy** [comment]
Great, I look forward to it.

**Comment by lucasecdb** [comment]
released in version v0.4.5!
